app/scripts/modules/core/src/pipeline/config/pipelineConfigurer.js

```diff
@@ -173,7 +174,8 @@ module.exports = angular.module('spinnaker.core.pipeline.config.pipelineConfigur
         controllerAs: '$ctrl',
         size: 'lg modal-fullscreen',
         resolve: {
-          pipeline: () => $scope.renderablePipeline,
+          pipeline: () => $scope.pipeline,
```

Comment:
This seems to revert the changes in https://github.com/spinnaker/deck/pull/4192, but I don't see how this PR addresses the bug that PR fixed. Can you verify that the behavior won't be reintroduced by this change?

Reply:
But did that PR really fix any bugs? It just changed the behavior of the edit JSON feature, and actually made it much harder to use templated pipelines. If you’d like to change the template after the change in #4192, you can’t do it in Deck anymore.
With this change, you can edit the configuration, and you can see (read only) the tendered pipeline. Best of both worlds!
But I’d like @chrisberry’s opinion on this, it was his PR.

Reply:
Yes - before that PR, changes in the pipeline JSON weren't reflected in the UI until you saved and refreshed.

I think you need `pipeline: () => $scope.isTemplatedPipeline ? $scope.pipeline : $scope.renderablePipeline`.

Reply:
But `$scope.renderablePipeline = $scope.plan || $scope.pipeline;`, so for non templated pipelines, this should not matter (because `$scope.plan` will be undefined). And for templated pipelines, it is just wrong. I don't want the renderable pipeline, I want the configuration, which is in `$scope.pipeline`.
I have tested, and the UI is updated when you do config changes, so I would argue that the change I introduce here is the correct behaviour.

Reply:
I've also tested the UI and it seems to work with this change, so I'm okay with it, even if I don't fully understand it (we are not using templates for the day-to-day management of Spinnaker internally).